composer.json

```diff
@@ -15,13 +15,13 @@
         }
     ],
     "require": {
-        "php": "^8.0",
+        "php": "^8.0|^8.1|^8.2",
```

Comment:
Does `^8.0` not imply 8.1 and 8.2 (and everything < 9.0)?

Reply:
Unfortunately, no. That was the root of this problem - I tried to `composer require` it and it said it only supported 8.0.